### PR TITLE
refactor: add-MovieHero-storybook

### PR DESCRIPTION
**Pull request by wolnio**

### CODE REVIEW

File: app/components/MovieHero/MovieHero.styles.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  flex-direction: column;
+  width: 50%;
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+`;
+
+const MediaBox = styled.div`
+  display: flex;
+  flex-direction: row;
```
Comment (marcelina-hasiak) [inline app/components/MovieHero/MovieHero.styles.js:34]: `flex-direction: row` is default when you declare flex, you can remove this line.

File: app/components/MovieHero/MovieHero.stories.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+const Template = (args) => <MovieHero {...args}></MovieHero>;
+
+export const Default = Template.bind({});
+Default.args = {
+  heroObj: [
+    {
+      poster_path: 'https://wallpaperaccess.com/full/2191678.jpg',
+      title: 'Whiplash',
+    },
+  ],
```
Comment (marcelina-hasiak) [inline app/components/MovieHero/MovieHero.stories.js:23]: The reason why background image  and title can't load inside our storybook is that you pass movie details object inside array.

**Review by marcelina-hasiak — CHANGES_REQUESTED** [review]

File: app/components/MovieHero/MuteIcon.js [hunk 3]
```diff
@@ -1,8 +1,24 @@
 import styled from 'styled-components';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
-export const MuteIcon = styled(FontAwesomeIcon)`
+export function MuteIcon(props) {
+  return (
+    <Button type="button" onClick={props.onClick} aria-pressed="false">
```
[the remark this answers is not in the document]
Comment (wolnio) [reply app/components/MovieHero/MuteIcon.js:6]: Oh, you're right, it's a testing residue. Aria was necessary to work with `play` function in StoryBook.

File: app/components/MovieHero/MediaButton.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
-export default function MediaButton(props) {
+export function MediaButton(props) {
```
[the remark this answers is not in the document]
Comment (wolnio) [reply app/components/MovieHero/MediaButton.js:4]: Well, when I add `default` I get a following error: `Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.`

[the remark this answers is not in the document]
Comment (marcelina-hasiak) [reply app/components/MovieHero/MediaButton.js:4]: Please remember to change import as well: `import MediaButton` not `import {MediaButton}`

**Review by marcelina-hasiak — APPROVED** [review]
1. Do you happen to know why this file is part of your PR?
Zrzut ekranu 2022-02-28 o 02 07 07
2. I assume you set this file in different PR? It needs to be changed to 'movie-hero'
Zrzut ekranu 2022-02-28 o 02 09 26  You can create a bug task in Jira and resolve it later on new branch.

Please investigate point 1. and change the export, apart from that everything looks good now ;)

**Pull request closed without merging**